tests/dump/bundled.wat.dump

```diff
@@ -126,13 +126,13 @@
 ============== func 1 ====================
    0x132 | 03          | size of function
    0x133 | 00          | 0 local blocks
-   0x134 | 00          | Unreachable
-   0x135 | 0b          | End
+   0x134 | 00          | op:unreachable
+   0x135 | 0b          | op:end
 ============== func 2 ====================
    0x136 | 03          | size of function
    0x137 | 00          | 0 local blocks
-   0x138 | 00          | Unreachable
-   0x139 | 0b          | End
+   0x138 | 00          | op:unreachable
+   0x139 | 0b          | op:end
```

Comment:
I feel like the `op:` prefix is going to get really repetitive with actaul wasms that we dump.